include/boost/url/url_base.hpp

```diff
@@ -1448,12 +1448,11 @@ class BOOST_SYMBOL_VISIBLE
     //--------------------------------------------
 
     friend
-    void
+    error_code
```

Comment:
Well this isn't the same as returning `result` now is it? Because the user has no easy way to convert the error into an exception. @pdimov ?

Reply:
result<void>

Reply:
Mmm... No problem. I didn't think `result<void>` would be desirable. I just never saw it.